test/tests/operations/LengthValueDecoder.mjs

```diff
@@ -0,0 +1,23 @@
+/**
+ * Length Value Decoder tests.
+ *
+ * @author gchq77703 []
+ * @copyright Crown Copyright 2018
+ * @license Apache-2.0
+ */
+
+import TestRegister from "../../TestRegister";
+
+TestRegister.addTests([
+    {
+        name: "KeyValue",
+        input: [5, 72, 111, 117, 115, 101, 4, 114, 111, 111, 109, 4, 100, 111, 111, 114],
```

Comment:
I can see your logic here and it should probably be possible to do, but as it stands, tests expect all input and output to be a string, in the same way that the web app takes all input as a string and returns all output as a string.

The data will be converted to a byteArray automatically by CyberChef before it is passed to the operation.

In this case, you probably want to pass in `"\x05\x48\x6f\x75\x73\x65\x04\x72\x6f\x6f\x6d\x04\x64\x6f\x6f\x72"` and expect `JSON.stringify([{"key": [25]..., null, 4)`.